docs/editor.md

```diff
@@ -33,37 +34,38 @@ const App = () => {
 
 ### `value: Value`
 
-this is the value that the editor should display. You will usually load that from a database.
-It has the type `Value`, but should generally considered to be "opaque", so don't rely on its internal structure.
+This is the content that the editor will display. The data for this prop may come from any source, eg a file or database or API etc. It is a JSON representation of the entire content that should be rendered, including the external visible information, and internal information required to render the information - eg ids, versions, plugin info, etc.
+
+This information is "opaque", and is not meant to be edited directly under normal circumstances, so don't rely on its internal structure.
+
+It does not contain any presentation aspects of the data, ie no CSS is stored.
 
 ### `onChange: (newValue: Value) => void`
 
-this function is called whenever the editor has new data. You would usually store the new data back to the database in this function.
-This is not required when `readOnly` is `true`
+A callback function whenever the editor has new data. This is not required when `readOnly` is `true`
```

Comment:
Maybe readd a sentence about storing the data

Reply:
I left that out on purpose. onChange is not equal to storing the data. When someone uses the editor, onChange will fire literally 100s of times or more. The onChange should update an internal variable, which is the standard controlled mode of react. At the end the user may discard changes, or save - and at that point the data would be stored somewhere, file, API etc, but that has nothing to do with the onChange callback.

Reply:
this is absolutly correct, yes.